webcompat/webhooks/helpers.py

```diff
@@ -18,6 +18,14 @@
 from webcompat.helpers import proxy_request
 
 BROWSERS = ['blackberry', 'brave', 'chrome', 'edge', 'firefox', 'iceweasel', 'ie', 'lynx', 'myie', 'opera', 'puffin', 'qq', 'safari', 'samsung', 'seamonkey', 'uc', 'vivaldi']  # noqa
+MOZILLA_BROWSERS = ['browser-fenix',
+                    'browser-firefox',
```

Comment:
This will fail when we separate firefox from the hierarchical structure.
[browser-firefox-ios](https://github.com/webcompat/web-bugs/issues?q=is%3Aissue+label%3Abrowser-firefox-ios) will become `browser-firefox` + `os-ios`.

Maybe we need to add a note on #2531 so we remember that to not break `engine-*`

Reply:
Done.